Project.toml

```
@@ -24,6 +25,7 @@ DataInterpolationsSymbolicsExt = "Symbolics"
 [compat]
 Aqua = "0.8"
 ChainRulesCore = "1"
+FindFirstFunctions = "1"
```

```suggestion
FindFirstFunctions = "1.1"
```

I think minimum compat for FindFirstFunctions.jl should be 1.1 - https://github.com/SciML/FindFirstFunctions.jl/pull/3